docs/general/quick-start/windows.md

```diff
@@ -0,0 +1,263 @@
+
+7. Allow Firewall Access when Prompted
+
+    ![Windows Firewall Prompt](/images/docs/quick-start/windows/jellyfin-4.png)
+
+    Windows Firewall Prompt
+
+8. Open a browser and go to [http://localhost:8096](http://localhost:8096) to finish the setup process. Leave the `Enable Automatic Port Mapping` option unchecked.
+
+### Hardware Accel
+
+See [Hardware Acceleration](/docs/general/administration/hardware-acceleration/)
+
+## Reverse Proxy
+
+### Router Settings
+
+1. Press `Win + R`, enter `control` and click `OK` to open Control Panel
+
+    ![Run Control](/images/docs/quick-start/windows/router-1.png)
+
+2. Navigate to `Network and Internet > Network and Sharing Center`
+3. Click on the `Internet Connection` under `Connections`
+
+    ![Ethernet Connection](/images/docs/quick-start/windows/router-2.png)
+
+4. Click `Details` in the popup and note down the `IPv4 Address` and `IPv4 Default Gateway` values. The `IPv4 Address` is the LAN IP address of your Jellyfin server and the `IPv4 Default Gateway` is usually your Router.
+
+    ![Network Connection Details](/images/docs/quick-start/windows/router-3.png)
+
+5. go to [https://ipv4.icanhazip.com](https://ipv4.icanhazip.com) and note down the IP address
+
+6. Go to the `IPv4 Default Gateway` IP in a browser (for this example, `http://192.168.50.1/`) and Login to your router. If you do not have the credentials, look for them on the bottom of your router or online with the model number of your router
+
+7. Most Routers will show a WAN IP in the index. Check if the WAN IP matches the IP you noted down in step 5. If it doesn't, you do not have a routable IP address. In this case, you should stop following this guide and look for other options for remote access.
+
+    ![ASUS router settings page](/images/docs/quick-start/windows/router-4.png)
+
+    ASUS Router Settings Page
+
+8. Assign your Jellyfin host a static local IP. The option is usually under `LAN` and named `DHCP Server` or `DHCP`. If you assign a different IP than what it currently has, you may need to reconnect the internet of this PC for settings to apply properly. In this case, I am assigning 192.168.50.142 to my PC.
+
+    ![Static DHCP](/images/docs/quick-start/windows/router-5.png)
+
+9. Find `Port Forwarding` settings in your router. This setting is usually under `WAN`, `Firewall`, `Security` or `NAT`and usually named `Port Forwarding` or `Virtual Server`. `Port Trigger` is NOT the setting you are looking for.
+
+10. Forward ports 80 and 443 to the IP you assigned previously
+
+    ![Port Forwarding](/images/docs/quick-start/windows/router-6.png)
+
+### Install Caddy
+
+1. Go to the [official Caddy server download page](https://caddyserver.com/download)
+2. Choose `Windows amd64` for the platform and click download. Leave the standard features checkbox checked. You do NOT need any extra features.
+
+    ![Caddy Download](/images/docs/quick-start/windows/caddyinst-1.png)
+
+3. Create a new folder under `C:\` and name it `caddy`
```

Comment:
I would highly suggest not adding a folder to your root. Something like `C:\Program Files\Caddy` would make more sense.